robottelo/test.py

```diff
@@ -9,6 +9,7 @@
 import os
 import pytest
 import unittest2
+from robottelo.ui.my_account import MyAccount
```

Comment:
This import should be moved to line 71 to keep the same pattern as the others ;)